apps/cpblscores/cpbl_scores.star

```diff
@@ -116,7 +114,7 @@ def main(config):
                 if NScount > NSMax:
                     continue
             gameTime = s.get("strTimestamp")
-            convertedTime = time.parse_time(gameTime.replace(":00+00:00", "Z"), format = "2006-01-02T15:04Z").in_location(timezone)
```

Comment:
this is intentional?

Reply:
Yes. Looks like the format returned by the API changed or something, but this wasn't working.